### DISCUSSION TITLE
Anybody seen this dialog box? Is the print sheet in place, empty and clean?

### DISCUSSION THREAD

**Discussion by Avpman2** [OrcaSlicer/OrcaSlicer#6823] [General]
All of a sudden,  I'm getting this pop-up dialog with most models. No matter which option I choose the print is NOT uploaded to the printer. No issues with Orca-Flashforge with the same models. I'm dead in the water as far as OrcaSlicer goes. Any help on what might be triggering the message would be greatly appreciated. TIA!

![os-error](https://github.com/user-attachments/assets/192c56a0-0fed-450e-bc2f-5d62db5aea2a)

**Comment by galosre** [comment]
Yes I am having this issue How did you fix it?

**Reply by dewi-ny-je** [reply]
Are we supposed to read your mind?

Start by providing configuration file, printer model, ....

No wonder no one answered in almost a year to the original question.
